src/CodeGen_X86.cpp

```diff
@@ -383,7 +383,8 @@ void CodeGen_X86::visit(const Max *op) {
 }
 
 void CodeGen_X86::visit(const Call *op) {
-    if (target.has_feature(Target::AVX2) &&
+    constexpr bool need_workaround = LLVM_VERSION < 40;
```

Comment:
I think constexpr requires msvc 2015 on windows. I believe we support one version earlier than that still. The following would be safer:

    if (LLVM_VERSION < 40 && target.has_feature...